compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -247,6 +247,11 @@ int entry(int argc, char **argv)
     .help("This will convert weight format of FullyConnected to SHUFFLED16x1FLOAT32. Note that "
           "it only converts weights whose row is a multiple of 16");
 
+  arser.add_argument("--replace_non_const_fc_with_batch_matmul")
+    .nargs(0)
+    .default_value(false)
+    .help("Replace Fully Connected with BatchMatMul when its weight is non-constant");
```

Comment:
maybe `Fully Connected` -> `FullyConnected` as it's operator type?
I didn't see this when I previously looked...

Reply:
```suggestion
    .help("Replace FullyConnected with BatchMatMul when its weight is non-constant");
```